workflow-examples/src/main/java/com/redhat/parodos/examples/move2kube/task/Move2KubeTransform.java

```diff
@@ -73,27 +73,28 @@ public WorkReport execute(WorkContext workContext) {
 			return new DefaultWorkReport(WorkStatus.FAILED, workContext, e);
 		}
 
-		String userId = String.valueOf(WorkContextUtils.getUserId(workContext));
-		if (!sendNotification(userId, workspaceId, projectId, transformId)) {
+		String message = sendNotification(workspaceId, projectId, transformId);
+		if (message == null) {
 			return new DefaultWorkReport(WorkStatus.FAILED, workContext,
 					new RuntimeException("Cannot notify user about the transformation status"));
 		}
-		return new DefaultWorkReport(WorkStatus.COMPLETED, workContext);
+		return new DefaultWorkReport(WorkStatus.COMPLETED, workContext, message);
```

Comment:
but if it's `COMPLETED`, then why should we display an alert message?

Reply:
this is the task that checker will check. and checker wont' be exposed to UI